### ISSUE TITLE
How to make a mesh label for semantic segmentation during training

### ISSUE THREAD

**Issue by 571502680** [cvlab-epfl/segmentation-driven-pose#6]

**Comment by YinlinHu** [comment]
2D reprojections of the mesh vertex according to the pose can tell you ground truth segmentations.

**Comment by 571502680** [comment]
But the paper uses the Semantic Separation Grid of S*S，if i use 2D reprojections of the mesh vertex, and its size is the shape of pictures!

**Comment by YinlinHu** [comment]
Just use the label of the nearest pixel, to generate a rough segmentation.

**Comment by 571502680** [comment]
Could you share the train.py?

**Comment by YinlinHu** [comment]
We will release an enhanced version of our method with the training code. (maybe in months)
